src/test/java/org/aszjch/demoapp/domain/article/port/KafkaTestListener.java

```diff
@@ -0,0 +1,15 @@
+package org.aszjch.demoapp.domain.article.port;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Component;
+
+@Component
+@Slf4j
+class KafkaTestListener {
+
+    @KafkaListener(topics = {"articles"})
```

Comment:
lepsza nazwa topika konieczna - articles-created np.
Nie musimy całej konwencji wymyślać, ale niech to będzie coś sensownego.